### ISSUE TITLE
Low Contrast - Web Content Accessibility Guidelines (WCAG 2.0)

### ISSUE THREAD

**Issue by alex-418** [BCcampus/tlpd#52]
@BCcLeva Just wanted to put this one on your radar for consideration. 
Disclaimer: I'm not an accessibility expert. 

Adequate contrast is necessary for all users, especially users with low vision. Currently, there appears to be very low contrast between foreground and background colors. The minimum level should be AA (as per [WCAG 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) contrast criteria). 

Report (click contrast tab):  http://wave.webaim.org/report#/https://tlpd.bccampus.ca/

![low contrast](https://user-images.githubusercontent.com/17072191/41440524-a3ca00a4-6fe3-11e8-9b32-fe296927825c.png)

**Comment by jgrajkowski** [comment]
Am happy to suggest a few shades of the existing colours we could change the menu and heading colours to, for next sprint. 

----- Original Message -----

**Comment by alex-418** [comment]
@jgrajkowski @BCcLeva to provide new colors

**Comment by jgrajkowski** [comment]
Header font colour options that provide better contrast but match well: purple (#8e53a1, matches purple in logo) or darker green (#116b44, matches some of the overlapping colours in logo) or deep blue (#2a5b75, matches some of the overlapping colours in logo)

**Comment by jgrajkowski** [comment]
is the blocker to decide on which? :)

**Comment by bdolor** [comment]
if the hex values for colours you supplied above are the ones we're going with, then the issue isn't blocked anymore. If there's still some decisions to be made then we'll wait until you're ready.

**Comment by jgrajkowski** [comment]
@BCcLeva to please choose and note here.  Blue or green would be my vote but I don't have a strong opinion between the two.

The colours were provided based on:
- The deep blue should match the text at the bottom of the logo ("...portal"); clean and professional
- Darker green just tweaks the current theme that includes a bit of green; fairly active colour
- Purple just 'cause it's dark/rich/jewel ;)

**Comment by BCcLeva** [comment]
Let's go for the darker green

**Comment by alex-418** [comment]
Completed via https://github.com/BCcampus/tlpd/commit/3270b40d0431dfb3920041187c323054be1cb55c please note, I used the dark green color of #006338 because it was very close to #116b44 which only met the AA standard. With this color we meet the AAA standard, here's the updated results from the report: 

![results](https://user-images.githubusercontent.com/17072191/45909306-ec8a0a80-bdb5-11e8-9d1e-6b79b017e72d.png)
